server/sonar-web/src/main/js/apps/users/components/GroupsForm.tsx

```diff
@@ -48,57 +47,34 @@ export default class GroupsForm extends React.PureComponent<Props> {
   }
 
   handleSearch = (query: string, selected: Filter) => {
-    return getUserGroups(this.props.user.login, undefined, query, selected).then(
-      (data: any) => {
-        this.setState({
-          groups: data.groups,
-          selectedGroups: data.groups
-            .filter((group: any) => group.selected)
-            .map((group: any) => group.name)
-        });
-      },
-      () => {}
-    );
+    return getUserGroups(this.props.user.login, undefined, query, selected).then((data: any) => {
+      this.setState({
+        groups: data.groups,
+        selectedGroups: data.groups
+          .filter((group: any) => group.selected)
```

Comment:
drop `any` type

Reply:
Did that for ProfileProject on ChangeProjectForm as well